docs/user/adding-glean-to-your-project.md

```diff
@@ -38,42 +38,59 @@ Add this to your Gradle configuration:
+In order for the Glean SDK to generate an API for your metrics, two Gradle plugins must be included in your build:
 
-```Groovy
-plugins {
-    id "com.jetbrains.python.envs" version "0.0.26"
-}
-```
+- The [Glean Gradle plugin](https://github.com/mozilla/glean/tree/master/gradle-plugin/)
+- JetBrains' [Python envs plugin](https://github.com/JetBrains/gradle-python-envs/)
 
-Right before the end of the same file, the Glean SDK build script must be included.
-This script can be referenced directly from the GitHub repo, as shown below:
+The Glean Gradle plugin is distributed on Maven, so we need to tell your build where to look for it by adding the following to the top of your `build.gradle`:
 
-```Groovy
-apply from: 'https://raw.githubusercontent.com/mozilla-mobile/android-components/v{latest-version}/components/service/glean/scripts/sdk_generator.gradle'
+```
+buildscript {
+    repositories {
+        maven {
+            url "https://maven.mozilla.org/maven2"
+        }
+
+        dependencies {
+            classpath "org.mozilla.telemetry:glean-gradle-plugin:{latest-version}"
+        }
+    }
+}
 ```
 
-> **Important:** the `{latest-version}` placeholder in the above link should be replaced with the version number of the Glean SDK used by the project.
-For example, if version *6.0.2* is used, then the include directive becomes:
+> **Important:** as above, the `{latest-version}` placeholder in the above link should be replaced with the version number of the Glean SDK used by the project.
+
+The JetBrains Python plugin is distributed in the Gradle plugin repository, so it can be included with:
 
 ```Groovy
-apply from: 'https://raw.githubusercontent.com/mozilla-mobile/android-components/v16.0.0/components/service/glean/scripts/sdk_generator.gradle'
+plugins {
+    id "com.jetbrains.python.envs" version "0.0.26"
+}
 ```
 
-If you are using a `SNAPSHOT` build of `android-components`, it is best to use the master branch, for example:
+Right before the end of the same file, we need to apply the Glean Gradle plugin:
 
 ```Groovy
-apply from: 'https://raw.githubusercontent.com/mozilla-mobile/android-components/master/components/service/glean/scripts/sdk_generator.gradle'
+apply plugin: "org.mozilla.telemetry.glean-gradle-plugin"
 ```
 
-There are [additional parameters](android-build-configuration-options.md) that can be set to control the behavior of the `sdk_generator.gradle` script, but they are rarely used in normal use.
+There are [additional parameters](android-build-configuration-options.md) that can be set to control the behavior of the Glean Gradle plugin, but they are rarely used in normal use.
+
+> **Note:** Earlier versions of Glean used a Gradle script (`sdk_generator.gradle`) rather than a Gradle plugin. Its use is deprecated and should projects should be updated to use the Gradle plugin as described above.
```

Comment:
```suggestion
> **Note:** Earlier versions of Glean used a Gradle script (`sdk_generator.gradle`) rather than a Gradle plugin. Its use is deprecated and projects should be updated to use the Gradle plugin as described above.
```